build.gradle

```diff
@@ -167,6 +167,7 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
+        maven { url = uri("http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/") }
```

Comment:
❓ @oconnormi should the above be http://nexus.phx.connexta.com:8081/nexus/content/groups/ion/?